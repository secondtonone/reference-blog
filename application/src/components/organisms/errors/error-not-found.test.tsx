import renderer from 'react-test-renderer';
import NotFound from './error-not-found';

jest.mock('~/components/containers/svg-icon', () => ({ code }) => <span>{code}</span>);

describe('Match snapshots ErrorNotFound', () => {
  it('404 page', () => {
    const tree = renderer.create(
      <NotFound />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
