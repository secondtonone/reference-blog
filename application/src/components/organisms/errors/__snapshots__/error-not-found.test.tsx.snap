// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots ErrorNotFound 404 page 1`] = `
<div
  className="box-adaptive__BoxStyled-sc-g2wy39-0 fiLzXF"
>
  <h2
    className="text-content__TextStyled-sc-1stkrsz-0 dRRiqw container__HeadingStyled-sc-orj98h-1 ievuLA"
    data-test="error-heading"
  >
    Sorry
  </h2>
  <div
    className="box-adaptive__BoxStyled-sc-g2wy39-0 kuJVxh container__MessageStyled-sc-orj98h-0 goEbG"
  >
    <p
      className="text-content__TextStyled-sc-1stkrsz-0 bUJWgm container__TextContentStyled-sc-orj98h-2 gWtmge"
      data-test="error-caption"
    >
      We can't find the page you're looking for.
    </p>
    <p
      className="text-content__TextStyled-sc-1stkrsz-0 bUJWgm container__TextContentStyled-sc-orj98h-2 gWtmge"
      data-test="error-text"
    >
      <article
        className="error-not-found__NotFoundStyled-sc-9vcyd6-0 jBoNKG"
      >
        You can visit ourÂ 
        <a
          data-test="error-link-main"
          href="/blog"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          homepage
        </a>
         
        or contact our
         
        <a
          data-test="error-link-support"
          href="mailto:mail@example.com?subject=example Blog"
          onClick={[Function]}
          onMouseEnter={[Function]}
          target="_blank"
        >
          support team
        </a>
        .
      </article>
    </p>
  </div>
</div>
`;
